apps/nextjs-website/src/components/organisms/WebinarQuestionsForm/WebinarQuestionsForm.tsx

```diff
@@ -53,6 +53,14 @@ export const WebinarQuestionsForm = ({
     setQuestion(e.target.value);
   };
 
+  const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (!question) return;
+    if (e.key === 'Enter' && (e.ctrlKey || e.shiftKey)) {
```

Comment:
I think you should explain why you chose this key combinations

Reply:
The design team chose to use the `shift+enter` combination of keys

Reply:
Maybe a comment on the function could be useful